src/app/pages/TransactionDetailPage/index.tsx

```diff
@@ -33,7 +38,9 @@ type TransactionSelectionResult = {
  *
  * Normally we want the successful one. If there is none, then the latest.
  */
-function useWantedTransaction(transactions: RuntimeTransaction[]): TransactionSelectionResult {
+function useWantedTransaction(transactionsList?: RuntimeTransactionList): TransactionSelectionResult {
```

Comment:
`transactionsList?: RuntimeTransactionList` is not the same as `transactionsList: RuntimeTransactionList | undefined`